.github/ISSUE_TEMPLATE/2-feature-request.yml

```diff
@@ -5,15 +5,14 @@ body:
     attributes:
       value: |
         Thank you for suggesting an idea to make Webiny better.
-        Please fill in as much of the template below as you're able.
+        Please fill in as much of the template below as you're able to.
   - type: textarea
     attributes:
       label: Is your feature request related to a problem? Please describe.
-      description: Please describe the problem you are trying to solve.
```

Comment:
Sorry @ArjunSharda, if it was not clear in my previous comment. I would like to keep `description` as well. Let's not remove it.

Reply:
Hi @ArjunSharda, it seems you mistakenly marked this comment as resolved. 
I am still able to see this line is deleted
> description: Please describe the problem you are trying to solve.

Could you please re-add it your repository, thank you!